rollup.config.js

```diff
@@ -14,8 +14,12 @@ export default {
 		{ file: pkg.main, 'format': 'umd', name }
 	],
 	plugins: [
-		svelte({ customElement: true, include: /\.wc\.svelte$/ }),
-		svelte({ customElement: false, exclude: /\.wc\.svelte$/ }),
+		svelte({ customElement: true, include: /\.wc\.svelte$/,
+			css: css => { css.write('dist/bundle.css'); }
+		}),
+		svelte({ customElement: false, exclude: /\.wc\.svelte$/,
+			css: css => { css.write('dist/bundle.css'); }
```

Comment:
actually I just noticed that this one is overwriting the bundle.css file generated per line 18. I don't know how to merge these styles

Reply:
what if I don't want to emit css file? we want to inline the styles in the style tag

Reply:
That would be great but I don't know how that can be done. This is sort of a hack I think. I'm only vaguely familiar with custom elements

Reply:
There is a kind of hack here: https://github.com/redradix/svelte-custom-element-template

But seems to be challenging to upgrade to the latest svelte loader...

If you find a way, I will be more than happy to use it 😄